### PR TITLE
PN-10290 added sp for all part shipped header status to shipped in so.

### PR DESCRIPTION
**Pull request by amitghediyasilverxis**

### CODE REVIEW

**Comment by MoinBloch1** [comment]
Please Add TRANSACTION

**Comment by amitghediyasilverxis** [comment]
> Please Add TRANSACTION

Done

**Comment by MoinBloch1** [comment]
Reviewed

File: PAS_Database/dbo/Stored Procedures/UpdateSalesOrderStatus.sql [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	BEGIN TRY
+		BEGIN TRANSACTION  
+			DECLARE @SoPartDataCount BIGINT,
+				@SoShippingCount BIGINT,
+				@SalesOrderShippingId BIGINT,
+				@SoShippingItemCount BIGINT,
+				@SOBillingInvoicingId BIGINT,
+				@SoBillingItemCount BIGINT;
+
+			SELECT @SoPartDataCount = COUNT([SalesOrderId]) FROM [DBO].[SalesOrderPartV1] WITH(NOLOCK) WHERE [SalesOrderId] = @SalesOrderId AND [IsActive] = 1 AND [IsDeleted] = 0;
```
Comment (rajeshgami93) [inline PAS_Database/dbo/Stored Procedures/UpdateSalesOrderStatus.sql]: IsACtive and IsDeleted check with ISNULL

**Review by rajeshgami93 — COMMENTED** [review]
Can you please add ISNull where you check IsDeleted and IsActive at all the places

**Review by vishalsutharsilverxis — APPROVED** [review]

**Pull request merged**